tests/InitCommandTest.php

```diff
@@ -0,0 +1,201 @@
+<?php
+
+namespace App\Tests;
+
+use phpmock\phpunit\PHPMock;
+
+class InitCommandTest extends TestCase
+{
+    use PHPMock;
+
+    public function setUp(): void
+    {
+        TestCase::setUp();
+    }
+
+    public function testRunWithoutAdminAndReadmeCreation()
+    {
+        $filePutContentsMock = $this->getFunctionMock('App\Console\Commands', 'file_put_contents');
+
+        $filePutContentsMock
+            ->expects($this->exactly(4))
+            ->withConsecutive(
+                ['.env.testing', $this->getFixture('env.testing.yml')],
+                ['.env', $this->getFixture('env.yml')],
+                ['.env.development', $this->getFixture('env.development.yml')],
+                ['.env.ci-testing', $this->getFixture('env.ci-testing.yml')],
+            );
```

Comment:
let's move it tot he separate helper `mockFilePutContent(array $fileContents)` of the separate mock trait `InitCommandMockTrait`

```php
$this-> mockFilePutContent([
    '.env.testing' => $this->getFixture('env.testing.yml'),
    '.env' => $this->getFixture('env.yml'),
    '.env.development' => $this->getFixture('env.development.yml'),
    '.env.ci-testing' => $this->getFixture('env.ci-testing.yml')
]);
```

Reply:
Done